### PR TITLE
Fix error introduced by latest rust: plugin syntax

### PR DESCRIPTION
**Pull request by toqueteos**
Build will fail until https://github.com/bjz/gl-rs/pull/305 get's merged.

### CODE REVIEW

**Comment by toqueteos** [comment]
Patch got merged, re-trying Travis.

File: src/main.rs [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 #![feature(collections, core, io, os, path, rustc_private, std_misc)]
 
-#[plugin]
+#![plugin(gfx_macros)]
 #[no_link]
 
 #[macro_use]
```
Comment (eddyb) [inline src/main.rs]: I don't think the `extern crate` needs to be here anymore.

Comment (toqueteos) [inline src/main.rs]: Compiler isn't complaining about extern crate...

Comment (tomaka) [inline src/main.rs]: You're importing an empty library with this `extern crate`. The compiler won't complain about it, but it's still unnecessary.

Comment (toqueteos) [inline src/main.rs]: `plugin` implies `extern crate` implicitly?

Comment (toqueteos) [inline src/main.rs]: Can't get rid of `extern gfx_macros`  (and `#[macro_use]`), otherwise `shaders!` macro isn't defined, just tried it.

Comment (tomaka) [inline src/main.rs]: Oh, apparently gfx-macros contains a regular macro: https://github.com/gfx-rs/gfx-rs/blob/master/src/gfx_macros/lib.rs#L167

I think that's an issue with gfx-macros.

**Comment by toqueteos** [comment]
This one didn't get merged and #194 did apply same changes so.. closing.

**Pull request closed without merging**